website/docs/d/database_user.html.markdown

```diff
@@ -63,6 +63,7 @@ In addition to all arguments above, the following attributes are exported:
 * `roles` - List of user’s roles and the databases / collections on which the roles apply. A role allows the user to perform particular actions on the specified database. A role on the admin database can include privileges that apply to the other databases as well. See [Roles](#roles) below for more details.
 * `x509_type` - X.509 method by which the provided username is authenticated.
 * `aws_iam_type` - The new database user authenticates with AWS IAM credentials. Default is `NONE`, `USER` means user has AWS IAM user credentials, `ROLE` - means user has credentials associated with an AWS IAM role.
+* `ldap_auth_type` - Method by which the provided username is authenticated.
```

Comment:
Some of these, like aws_iam_type have the accepted types and some don't.  Let's include them for ldap here and we'll update the others later - they are at https://docs.atlas.mongodb.com/reference/api/database-users-create-a-user